SignalServiceKit/src/Loki/Database/OWSPrimaryStorage+Loki.h

```diff
@@ -8,6 +8,21 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSInteger, LKFriendRequestStatus) {
```

Comment:
Weren't we going to consolidate this down to fewer statuses? Fine to do that later; just wondering.

Also, when is the last time we tested friend request expiration? 🤔

Reply:
The statuses would remain the same. We were going to reduce down the duplicated statuses (Thread FR, Message FR)

Don't think we have tested it thoroughly